frontend/src/metabase/hoc/UserHasSeen/UserHasSeen.mdx

```diff
@@ -0,0 +1,12 @@
+# UserHasSeen
```

Comment:
These aren't HOCs, please don't put them in `/hoc/` dir :)

Also: HOCs are deprecated, see https://github.com/metabase/metabase/pull/30986#issuecomment-1563061794.

Reply:
Done :)